src/components/containers/ConnectedMellowLpDepositForm/ConnectedMellowLpDepositForm.tsx

```diff
@@ -0,0 +1,221 @@
+        hintText: "Initialising, please wait",
+    }
+
+    switch (depositState) {
+        case DepositStates.INITIALISING: {
+            submissionState = {
+                submitText: "Initialising...",
+                action: () => { },
+                hintText: "Initialising, please wait",
+                disabled: true,
+            }
+            break;
+        }
+        case DepositStates.PROVIDER_ERROR: {
+            submissionState = {
+                submitText: "Provider Error",
+                action: () => { },
+                hintText: error,
+                disabled: true,
+            }
+            break;
+        }
+        case DepositStates.APPROVE_FAILED: {
+            submissionState = {
+                submitText: "Approve",
+                action: approve,
+                hintText: error,
+                disabled: false,
+            }
+            break;
+        }
+        case DepositStates.DEPOSIT_FAILED: {
+            submissionState = {
+                submitText: "Deposit",
+                action: deposit,
+                hintText: error,
+                disabled: false,
+            }
+            break;
+        }
+        case DepositStates.APPROVE_REQUIRED: {
+            submissionState = {
+                submitText: "Approve",
+                action: approve,
+                hintText: `Please approve ${vault.tokenName}`,
+                disabled: false,
+            }
+            break;
+        }
+        case DepositStates.APPROVING: {
+            submissionState = {
+                submitText: "Approving...",
+                action: () => { },
+                hintText: "Waiting for confirmation",
+                disabled: true,
+            }
+            break;
+        }
+        case DepositStates.APPROVED: {
+            submissionState = {
+                submitText: "Deposit",
+                action: deposit,
+                hintText: (selectedDeposit > 0) ? (<><Text green>Tokens approved</Text>. Let's deposit!</>) : "Please input amount",
+                disabled: !(selectedDeposit > 0),
+            }
+            break;
+        }
+        case DepositStates.DEPOSITING: {
+            submissionState = {
+                submitText: "Depositing...",
+                action: () => { },
+                hintText: "Waiting for confirmation",
+                disabled: true,
+            }
+            break;
+        }
+        case DepositStates.DEPOSIT_DONE: {
+            submissionState = {
+                submitText: "Deposit",
+                action: deposit,
+                hintText: (<Text green>Deposited</Text>),
+                disabled: false,
+            }
+            break;
+        }
+    }
+
+    const handleGoBack = () => {
+        onReset();
+        navigate(`/${routes.ECOSYSTEM}`);
+    }
+
+    const onChangeDeposit = (value: number | undefined): void => {
+        setSelectedDeposit(value ?? 0);
+    }
+
+    return (
+        <>
```

Comment:
I cannot see why you used <> </> here. Seems redudant

Reply:
Resolved. https://github.com/Voltz-Protocol/voltz-ui/pull/295/commits/a44c3c48e2a2660e1fc14f6eeeda646152d2edea